### PR TITLE
Enable ppc64le in echo-api

### PR DESCRIPTION
**Pull request by krishvoor**
This PR enables ppc64le architecture in echo-api #211 
Travis job is triggered for every PR submitted

Signed-off-by: Krishna Harsha Voora <krishvoor@in.ibm.com>

### CODE REVIEW

**Comment by krishvoor** [comment]
Hi @unleashed - Request for your review ^^^

**Comment by unleashed** [comment]
Looks good, I'll enable the Travis integration and check the test run.

File: .travis/build_ppc64le.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ev
+
+# Start Echo API
+cd ..
+bundle exec rackup --port 3000 --host 0.0.0.0 --env production &
```
Comment (krishvoor) [inline .travis/build_ppc64le.sh:7]: Note to self - I think the job would fail here :-(

Comment (unleashed) [reply .travis/build_ppc64le.sh:7]: Ok, guess we'll find out soon.

**Review by unleashed — APPROVED** [review]

**Comment by unleashed** [comment]
This is the failed build log:

```
Bundle complete! 9 Gemfile dependencies, 19 gems now installed.

Use `bundle info [gemname]` to see where a bundled gem is installed.

10.04s$ .travis/build_${TRAVIS_CPU_ARCH}.sh

# Start Echo API

cd ..

bundle exec rackup --port 3000 --host 0.0.0.0 --env production &

# Sleep

sleep 10

configuration /home/travis/build/3scale/config.ru not found

# test Echo API

curl --fail http://0.0.0.0:3000/test

curl: (7) Failed to connect to 0.0.0.0 port 3000: Connection refused

The command ".travis/build_${TRAVIS_CPU_ARCH}.sh" exited with 7.

Done. Your build exited with 1.
```

File: .travis/build_ppc64le.sh (hunk 1 above)
Comment (krishvoor) [reply .travis/build_ppc64le.sh:7]: My guess was right, I will push few changes on top of this.
`configuration /home/travis/build/3scale/config.ru not found`

**Pull request merged**